.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: false
 
 env:
   global:
-    - CONDA_DEPS="pip flake8 numpy scipy numba joblib dask" PIP_DEPS="pytest coveralls pytest-cov"
+    - CONDA_DEPS="pip flake8 numpy scipy numba joblib dask nose" PIP_DEPS="pytest coveralls pytest-cov"
```

Comment:
Gotcha - makes sense. We might need to eventually shift to using the pytest error assertions. Something like this, I believe: http://stackoverflow.com/questions/23337471/how-to-properly-assert-that-exception-raises-in-pytest